### ISSUE TITLE
What do you think about?

### ISSUE THREAD

**Issue by Durkee** [fingers10/BlazorParallax#3]
What do you think about?

1.  Create a view with a Parallax scroll effect.  e.g. https://materializecss.com/parallax-demo.html
2.  Add a new project for Blazor WebAssembly
3.  Port Materializecss Carousel to Blazor https://materializecss.com/carousel.html  (it's a slow page.... wait for it)

Cheers,
Steve

**Comment by fingers10** [comment]
Yeah we can give a try. Your working on this? Any PR is most welcomed
